recipe/meta.yaml

```diff
@@ -10,17 +10,18 @@ source:
   sha256: b068d6446ab7c227f703e4670862274ce3eea14e2925a41eae48c184fdf32ce2
 
 build:
+  noarch: python
   number: 0
```

Comment:
```suggestion
  number: 1
```

Reply:
The build number is for changes like this

We may also need to wait for 2.0.2 since we're removing pickle5 there.

Reply:
Yeah I realized 2.0.2 was not updated yet, adding in those changes now